doc/api/sqlite.md

```diff
@@ -234,10 +234,26 @@ added:
 * `options` {Object} The configuration options for how the changes will be applied.
   * `filter` {Function} Skip changes that, when targeted table name is supplied to this function, return a truthy value.
     By default, all changes are attempted.
-  * `onConflict` {number} Determines how conflicts are handled. **Default**: `SQLITE_CHANGESET_ABORT`.
-    * `SQLITE_CHANGESET_OMIT`: conflicting changes are omitted.
-    * `SQLITE_CHANGESET_REPLACE`: conflicting changes replace existing values.
-    * `SQLITE_CHANGESET_ABORT`: abort on conflict and roll back database.
+  * `onConflict` {Function} A function that determines how to handle conflicts. The function receives one argument,
```

Comment:
Nit: Would prefer `onconflict` here instead of the camel-case `onConflict` just to make things slightly easier when typing.

Reply:
I can't find of any method in the Node.js API that takes an object that does not use camel case. I think it's better to use camel case for consistency.

https://nodejs.org/api/tls.html#new-tlstlssocketsocket-options

https://nodejs.org/api/http.html#new-agentoptions

https://nodejs.org/docs/latest/api/http.html#httprequestoptions-callback

What do you think?